discovery-provider/Dockerfile

```diff
@@ -107,7 +107,7 @@ ENV audius_loggly_tags=$audius_loggly_tags
 ENV PROMETHEUS_MULTIPROC_DIR /prometheus_data
 RUN mkdir -p ${PROMETHEUS_MULTIPROC_DIR}
 
-HEALTHCHECK --interval=5s --timeout=5s \
-    CMD curl -f http://localhost:5000/health_check || exit 1
+HEALTHCHECK --interval=5s --timeout=5s --retries=12 \
```

Comment:
just so i understand whats happening here - this will try to healthcheck the container 12 times in 5 second intervals unless alembic is running. once alembic stops running it'll go back to the regular healthcheck url

Reply:
Yes, it goes back to regular health check after alembic.

As long as there is at least one success in the 12 checks, this will return a positive.

Reply:
got it, so if there's 12 negative retries after 65 seconds (5 second initial delay + (5sec*12 retries)) it restarts the pod?